packages/shared/src/libs/helpers/get-formatted-date/get-formatted-date.helper.ts

```diff
@@ -0,0 +1,9 @@
+import { format } from "date-fns";
+
+const getFormattedDate = (): string => {
```

Comment:
Send format as an argument

Reply:
Should I change the location of the "date-fns" package? (It's currently in dev dependencies in shared)

Reply:
It is not a dev dependency, we need this package in production. Should be in dependencies in shared